# Add after line 36 (set(PROJECT_VERSION "0.1.0"))
set(CLIENT_VERSION_MAJOR 1)
set(CLIENT_VERSION_MINOR 0)
set(CLIENT_VERSION_REVISION 0)
set(CLIENT_VERSION_BUILD 0)
set(CLIENT_VERSION_IS_RELEASE 0)
set(BUILD_DATE "${BUILD_TIME}")
set(GIT_COMMIT_ID "${GIT_HASH}")

# Fix configure_file line (around line 39)
configure_file(${CMAKE_SOURCE_DIR}/include/compat/version_config.h.in
               ${CMAKE_BINARY_DIR}/version_config.h @ONLY)

# Add include directory for generated version
include_directories(${CMAKE_BINARY_DIR})
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3195,6 +3195,7 @@
   target_include_directories(test_header_sync_restart PRIVATE
     ${CMAKE_SOURCE_DIR}/include
     ${CMAKE_SOURCE_DIR}/src
+    ${CMAKE_SOURCE_DIR}/third_party/snappy/third_party/googletest/googletest/include
   )
   if(APPLE)
     target_link_libraries(test_header_sync_restart PRIVATE
